Library/Formula/grafana.rb

```diff
@@ -0,0 +1,92 @@
+require "language/go"
+
+class Grafana < Formula
+  homepage "http://www.grafana.org/"
+  head "https://github.com/grafana/grafana.git"
+  version "2.0.0-beta1"
+
+  stable do
+    url "https://github.com/grafana/grafana/archive/v2.0.0-beta1.tar.gz"
```

Comment:
Yes but they either need a stable block too or need to start in the `homebrew-devel-only` tap.